sympy/core/tests/test_wester.py

```diff
@@ -924,56 +929,67 @@ def test_M12():
     ]
 
 
+@XFAIL
 def test_M13():
-    assert solve(sin(x) - cos(x), x) == [-3*pi/4, pi/4]
+    n = Dummy('n')
+    assert solveset_real(sin(x) - cos(x), x) == ImageSet(Lambda(n, n*pi - 7*pi/4), S.Integers)
 
 
+@XFAIL
 def test_M14():
-    assert solve(tan(x) - 1, x) == [pi/4]
+    n = Dummy('n')
+    assert solveset_real(tan(x) - 1, x) == ImageSet(Lambda(n, n*pi + pi/4), S.Integers)
 
 
+@XFAIL
 def test_M15():
-    assert solve(sin(x) - S.Half) == [pi/6, 5*pi/6]
+    n = Dummy('n')
+    assert solve(sin(x) - S.Half) == Union(ImageSet(Lambda(n, 2*n*pi + pi/6), S.Integers),
+                                           ImageSet(Lambda(n, 2*n*pi + 5*pi/6), S.Integers))
 
 
+@XFAIL
 def test_M16():
-    assert solve(sin(x) - tan(x), x) == [0, -pi, pi, 2*pi]
+    n = Dummy('n')
+    assert solve(sin(x) - tan(x), x) == ImageSet(Lambda(n, n*pi), Integers())
 
 
 @XFAIL
 def test_M17():
-    assert solve(asin(x) - atan(x),x) == [0]
+    assert solveset_real(asin(x) - atan(x), x) == FiniteSet(0)
```

Comment:
There should be a white space after a comma, earlier it wasn't so I had put that.

If I would remove that, the PEP8 error would be:
`missing whitespace after ','`